packages/blocks/src/api/factory.js

```diff
@@ -402,22 +444,7 @@ export function switchToBlockType( blocks, name ) {
 		return null;
 	}
 
-	let transformationResults;
-
-	if ( transformation.isMultiBlock ) {
-		if ( has( transformation, '__experimentalConvert' ) ) {
-			transformationResults = transformation.__experimentalConvert( blocksArray );
-		} else {
-			transformationResults = transformation.transform(
-				blocksArray.map( ( currentBlock ) => currentBlock.attributes ),
-				blocksArray.map( ( currentBlock ) => currentBlock.innerBlocks ),
-			);
-		}
-	} else if ( has( transformation, '__experimentalConvert' ) ) {
```

Comment:
This is in the stable product a resolution of  #40316 will be needed prior to merge.